src/watch/index.ts

```diff
@@ -145,22 +157,25 @@ export const watchCommand = command({
 	 * As an alternative, we watch cwd and all run-time dependencies
 	 */
 	const watcher = watch(
-		argv._,
+		[...argv._, ...argv.flags.include],
 		{
 			cwd: process.cwd(),
 			ignoreInitial: true,
-			ignored: [
-				// Hidden directories like .git
-				'**/.*/**',
+			ignored: !argv.flags.noIgnore
```

Comment:
I think this is a pretty big foot-gun that leads devs into watching a ton of files.

Supporting watching `.env` is a good example, but we'll need to think of a better way to make these exceptions. Ideally via `--include`.

I don't want it to block this PR so can you file a new issue for discussion or open a PR with a proposal?

Reply:
Ok, I have removed `ignored` tag and used `include` to filter the list of `ignore`.

Reply:
Currently, I don't have the capacity to test and review that functionality.

I don't want it to block this PR so can you file a new issue for discussion or open a PR with a proposal?

Reply:
I updated this pr and opened a new issue https://github.com/esbuild-kit/tsx/issues/210.